.gitmodules

```diff
@@ -5,3 +5,7 @@
 	path = deps/ua-nodeset
 	url = https://github.com/OPCFoundation/UA-Nodeset
 	branch = latest
+[submodule "deps/mqtt-c"]
+	path = deps/mqtt-c
+	url = https://github.com/LiamBindle/MQTT-C.git
+	branch = master
```

Comment:
would it make sense/more stable to pin to releases?

and then perhaps utilize dependabot to check for new releases?

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

Reply:
The repo should be set to the v1.1.6 tag or its commit. However, this is not visible in the .gitsubmodules file.